### PR TITLE
Add hardcoded "comments" in the form of JSON

### PR DESCRIPTION
**Pull request by maceonthompson**

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -19,14 +19,26 @@
...
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import java.util.Arrays;
+import com.google.gson.Gson;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private ArrayList<String> messages = new ArrayList<String>(Arrays.asList("msg1", "msg2", "msg3"));
```
Comment (FabriceCastel) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:30]: You probably don't need to use `ArrayList` in this code. You could replace it with `List`, which is the type that `Arrays.asList` returns. In general, you shouldn't need to reach for a specific implementation of the `List` interface unless you need to instantiate a `List` yourself.

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -40,12 +40,24 @@ function redirectToRandomSong(){
 }
 
 /**
-  Fetches the (currently hardcoded) greeting from the server and adds it to the DOM.
+ * Fetches (currently hardcoded) comments and prints them to the console.
```
Comment (FabriceCastel) [inline portfolio/src/main/webapp/script.js:43]: Is this comment still accurate? Looks like the code still adds these elements to the DOM

Comment (maceonthompson) [reply portfolio/src/main/webapp/script.js:43]: My bad, that was supposed to be updated! It now adds the commments (vs the "greeting") to the DOM.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 3]
```diff
@@ -27,7 +27,7 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  private ArrayList<String> messages = new ArrayList<String>(Arrays.asList("msg1", "msg2", "msg3"));
+  private List<String> messages = new ArrayList<String>(Arrays.asList("msg1", "msg2", "msg3"));
```
Comment (FabriceCastel) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:30]: Pretty sure you can replace replace `new ArrayList<String>(Arrays.asList("msg1", "msg2", "msg3"))` with just the `Arrays.asList("msg1", "msg2", "msg3")` part and remove the corresponding ArrayList import

**Pull request merged**